gobblin-modules/gobblin-kafka-09/src/main/java/org/apache/gobblin/kafka/writer/Kafka09DataWriter.java

```diff
@@ -126,4 +138,33 @@ public void flush()
       throws IOException {
     this.producer.flush();
   }
+  
+  private void provisionTopic(String topicName,Config config) {
+		String zooKeeperPropKey = KafkaWriterConfigurationKeys.KAFKA_TOPIC_CONFIG+KafkaWriterConfigurationKeys.CLUSTER_ZOOKEEPER;
+		if(!config.hasPath(zooKeeperPropKey)){
+		    LOG.debug("Topic "+topicName+" is configured without the partition and replication");
+			return;
+		}
+	    String zookeeperConnect = config.getString(zooKeeperPropKey);
+	    int sessionTimeoutMs = 10 * 1000;
+	    int connectionTimeoutMs = 8 * 1000;
+	    // Note: You must initialize the ZkClient with ZKStringSerializer.  If you don't, then
+	    // createTopic() will only seem to work (it will return without error).  The topic will exist in
+	    // only ZooKeeper and will be returned when listing topics, but Kafka itself does not create the
+	    // topic.
+	    ZkClient zkClient = new ZkClient(zookeeperConnect,sessionTimeoutMs,connectionTimeoutMs,ZKStringSerializer$.MODULE$);
+	    // Security for Kafka was added in Kafka 0.9.0.0
+	    ZkUtils zkUtils = new ZkUtils(zkClient, new ZkConnection(zookeeperConnect), false);
+	    String partitionPropKey = KafkaWriterConfigurationKeys.KAFKA_TOPIC_CONFIG+KafkaWriterConfigurationKeys.PARTITION_COUNT;
+	    String replicationPropKey = KafkaWriterConfigurationKeys.KAFKA_TOPIC_CONFIG+KafkaWriterConfigurationKeys.REPLICATION_COUNT;
+	    int partitions = ConfigUtils.getInt(config,partitionPropKey,KafkaWriterConfigurationKeys.PARTITION_COUNT_DEFAULT);
```

Comment:
Please add a single space after comma. It might help to use a code style file in the IDE